### DISCUSSION TITLE
Sharing a single Genshin Impact installation between multiple macOS users

### DISCUSSION THREAD

**Discussion by taupter** [PlayCover/PlayCover#101] [Q&A] [answered]
Hello. I have a question. I have a Macbook Air M1, with PlayCover 1.0.7 installed and Genshin Impact 2.8.0 running on it. I play the game, but my daughter wants to play too on the same Mac. Due to the problems we already know, we need to do all that reboot magic to log in GI, but every time we need to change the GI user it needs to go through the reboot/safety mode cycle. One alternative I was contemplating was to make her use GI on her Mac's user account, but it would imply reinstalling and redownloading everything, making two copies of the full GI data on the space-starving internal SSD (each one in its own /Users/<user>/Library/Containers/com.miHoYo.GenshinImpact). Is there a way to download/install GI to a shared folder, so two different users could enjoy playing GI on their respective user accounts, without doubling the download and storage?

**Comment by Depal1** [comment, marked as answer]
Hello! An account switching feature is being worked on. It will allow you to switch accounts under the same Mac user without the tedious rebooting procedure.

**Comment by Depal1** [comment]
The feature is now on a nightly release if you want to test it. https://github.com/PlayCover/PlayCover/actions/runs/2762881043

**Reply by Depal1** [reply]
It might be unstable though.

**Reply by taupter** [reply]
I did the following: restarted in safe mode, csrutil disable, rebooted, logged into GI. Rebooted in safe mode, csrutil enable, rebooted, tested 1st account, ok.
Then restarted in safe mode, csrutil disable, rebooted, logged off GI and logged into with second account. Rebooted in safe mode, csrutil enable, rebooted, tested 2nd account, ok. Logged off GI, tested 1st account, ok. Logged off, logged with 2nd account, ok.
Apart from the pain of doing the procedure again every time a new account is added or a password is changed, now I can change counts without a fuss. Yay! Thanks!
